pkg/workload/tpcc/tpcc.go

```diff
@@ -264,6 +268,8 @@ var tpccMeta = workload.Meta{
 		g.flags.BoolVar(&g.replicateStaticColumns, `replicate-static-columns`, false, "Create duplicate indexes for all static columns in district, items and warehouse tables, such that each zone or rack has them locally.")
 		g.flags.BoolVar(&g.localWarehouses, `local-warehouses`, false, `Force transactions to use a local warehouse in all cases (in violation of the TPC-C specification)`)
 		g.flags.StringVar(&g.queryTraceFile, `query-trace-file`, ``, `File to write the query traces to. Defaults to no output`)
+		g.flags.Uint32Var(&g.fakeTime, `fake-time`, 0, `If set, randomly generate times starting from this unix epoch for queries instead of using the current time`)
+		g.flags.Uint32Var(&g.fakeTimeStep, `fake-time-step`, 0, `If set, increment the fake time by a random value in seconds up to this step value after each query`)
```

Comment:
Do we have a use-case for this feature? I'm wondering if it wouldn't be sufficient to keep simple and have workload itself define a max step.

Reply:
Yeah that's fair -- it's not really needed